api/handler/acl.go

```diff
@@ -167,6 +167,8 @@ func (h *handler) GetBucketACLHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	h.log.Debug("target details", zap.String("container", reqInfo.BucketName), zap.Stringer("cid", bktInfo.CID))
```

Comment:
I would use the `bucket` instead of `container`

Reply:
Ok, will fix that too.